packages/headless/package.json

```diff
@@ -8,22 +8,148 @@
+      "node": {
+        "types": "./dist/definitions/index.d.ts",
+        "import": "./dist/headless.esm.js",
+        "require": "./dist/headless.js"
+      },
+      "browser": {
+        "types": "./dist/definitions/index.d.ts",
+        "import": "./dist/browser/headless.esm.js",
+        "require": "./dist/browser/headless.js"
+      },
+      "import": "./dist/browser/headless.esm.js",
+      "require": "./dist/browser/headless.js"
+    },
+    "./case-assist": {
+      "types": "./dist/definitions/case-assist.index.d.ts",
+      "node": {
+        "types": "./dist/definitions/case-assist.index.d.ts",
+        "import": "./dist/case-assist/headless.esm.js",
+        "require": "./dist/case-assist/headless.js"
+      },
+      "browser": {
+        "types": "./dist/definitions/case-assist.index.d.ts",
+        "import": "./dist/browser/case-assist/headless.esm.js",
+        "require": "./dist/browser/case-assist/headless.js"
+      },
+      "import": "./dist/browser/case-assist/headless.esm.js",
+      "require": "./dist/browser/case-assist/headless.js"
+    },
+    "./commerce": {
+      "types": "./dist/definitions/commerce.index.d.ts",
+      "node": {
+        "types": "./dist/definitions/commerce.index.d.ts",
+        "import": "./dist/commerce/headless.esm.js",
+        "require": "./dist/commerce/headless.js"
+      },
+      "browser": {
+        "types": "./dist/definitions/commerce.index.d.ts",
+        "import": "./dist/browser/commerce/headless.esm.js",
+        "require": "./dist/browser/commerce/headless.js"
+      },
+      "import": "./dist/browser/commerce/headless.esm.js",
+      "require": "./dist/browser/commerce/headless.js"
+    },
+    "./insight": {
+      "types": "./dist/definitions/insight.index.d.ts",
+      "node": {
+        "types": "./dist/definitions/insight.index.d.ts",
+        "import": "./dist/insight/headless.esm.js",
+        "require": "./dist/insight/headless.js"
+      },
+      "browser": {
+        "types": "./dist/definitions/insight.index.d.ts",
+        "import": "./dist/browser/insight/headless.esm.js",
+        "require": "./dist/browser/insight/headless.js"
+      },
+      "import": "./dist/browser/insight/headless.esm.js",
+      "require": "./dist/browser/insight/headless.js"
+    },
+    "./product-listing": {
+      "types": "./dist/definitions/product-listing.index.d.ts",
+      "node": {
+        "types": "./dist/definitions/product-listing.index.d.ts",
+        "import": "./dist/product-listing/headless.esm.js",
+        "require": "./dist/product-listing/headless.js"
+      },
+      "browser": {
+        "types": "./dist/definitions/product-listing.index.d.ts",
+        "import": "./dist/browser/product-listing/headless.esm.js",
+        "require": "./dist/browser/product-listing/headless.js"
+      },
+      "import": "./dist/browser/product-listing/headless.esm.js",
+      "require": "./dist/browser/product-listing/headless.js"
+    },
+    "./product-recommendation": {
```

Comment:
Those two are going away

Reply:
Right, ill wait for the merge and update this branch